examples/cd_distillation_cifar10.ipynb

```diff
@@ -0,0 +1,781 @@
+    "\n",
+    "We can leverage the scores obtained trained on normal data and employ model distillation as a data drift detector. The score function of the detector becomes the preprocessing function for the drift detector. The K-S test is then a simple univariate test on the harmfulness scores. Importantly, an harmfulness detector flags **malicious data drift**. We can fetch the pretrained model distillation detector from a [Google Cloud Bucket](https://console.cloud.google.com/storage/browser/seldon-models/alibi-detect/ad/cifar10/resnet32) or train one from scratch:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Definition and distillation of the ancillary model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ancilla_model_cifar10(clf,\n",
+    "                          nb_conv_layers=3,\n",
+    "                          nb_filters1=256,\n",
+    "                          nb_dense=40,\n",
+    "                          kernel1=4,\n",
+    "                          kernel2=4,\n",
+    "                          kernel3=4,\n",
+    "                          ae_arch=False):\n",
+    "\n",
+    "    print('Define ancillary model')\n",
+    "    nb_filters1 = int(nb_filters1)\n",
+    "    nb_filters2 = int(nb_filters1 / 2)\n",
+    "    nb_filters3 = int(nb_filters1 / 4)\n",
+    "    layers = [InputLayer(input_shape=(32, 32, 3)),\n",
+    "              Conv2D(nb_filters1, kernel1, strides=2, padding='same')]\n",
+    "    if nb_conv_layers > 1:\n",
+    "        layers.append(Conv2D(nb_filters2, kernel2, strides=2, padding='same',\n",
+    "                             activation=tf.nn.relu, kernel_regularizer=l1(1e-5)))\n",
+    "    if nb_conv_layers > 2:\n",
+    "        layers.append(Conv2D(nb_filters3, kernel3, strides=2, padding='same',\n",
+    "                             activation=tf.nn.relu, kernel_regularizer=l1(1e-5)))\n",
+    "    layers.append(Flatten())\n",
+    "    layers.append(Dense(nb_dense))\n",
+    "    layers.append(Dense(clf.output_shape[1], activation='softmax'))\n",
+    "    ancillary_model = tf.keras.Sequential(layers)\n",
+    "\n",
+    "    return ancillary_model\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def accuracy(y_true: np.ndarray, y_pred: np.ndarray) -> float:\n",
+    "    return (y_true == y_pred).astype(int).sum() / y_true.shape[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "load_pretrained = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filepath = '/home/fdz/.keras/datasets/' # change to (absolute) directory where model is downloaded\n",
```

Comment:
Best not to hardcode custom paths. Alternative: `filepath = os.path.join(os.getcwd(), 'model_distillation')`